src/components/Workflow/__stories__/SearchableWorkflowNameList.stories.tsx

```diff
@@ -7,4 +8,10 @@ const baseProps = { workflows: [...sampleWorkflowNames] };
 
 const stories = storiesOf('Workflow/SearchableWorkflowNameList', module);
 stories.addDecorator((story) => <div style={{ width: '650px' }}>{story()}</div>);
-stories.add('basic', () => <SearchableWorkflowNameList {...baseProps} />);
+stories.add('basic', () => (
+  <SearchableWorkflowNameList
```

Comment:
did it work for you in `yarn storybook`?
I think you need to wrap it up with SnackBarProvider see src/components/Workflow/__stories__/SearchableWorkflowNameList.stories.tsx
Feel free to separate Wrapper as new export into `src/components/common/__stories__/Decorators.tsx`  or add SnackBarProvider to  StorybookContainer